internal/installation/stop.go

```diff
@@ -35,5 +60,53 @@ func StopRunning(installPath string) (rerr error) {
 		}
 	}
 
+	procs, err := process.Processes()
+	if err != nil {
+		return errs.Wrap(err, "Could not get list of running processes")
+	}
+
+	// This is a bit heavy handed but ensures that there are
+	// no running state-svc processes which could lead to
+	// errors when updating
+	for _, p := range procs {
+		n, err := p.Name()
+		if err != nil {
+			logging.Error("Could not get process name: %v", err)
+			continue
+		}
+
+		if n == constants.ServiceCommandName {
```

Comment:
Can we get the full path here? I know it's very unlikely that there are other processes named `state-svc`, but it would be nice if we could have an extra safety check here. eg. check that `activestate` is in the path.